components/TextFields/examples/TextFieldObjectiveCExample.m

```diff
@@ -0,0 +1,353 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MaterialTextFields.h"
+
+@interface TextFieldOutlinedObjectiveCExample : UIViewController <UITextFieldDelegate, UITextViewDelegate>
+
+@property(nonatomic) MDCTextInputControllerOutlinedField *nameController;
+@property(nonatomic) MDCTextInputControllerOutlinedField *addressController;
+@property(nonatomic) MDCTextInputControllerOutlinedField *cityController;
+@property(nonatomic) MDCTextInputControllerOutlinedField *stateController;
+@property(nonatomic) MDCTextInputControllerOutlinedField *zipController;
+@property(nonatomic) MDCTextInputControllerOutlinedField *phoneController;
+
+@property(nonatomic) MDCTextInputControllerTextArea *messageController;
+
+@property(nonatomic) UIScrollView *scrollView;
+
+@end
+
+@implementation TextFieldOutlinedObjectiveCExample
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  self.view.backgroundColor = [UIColor whiteColor];
+
+  [self registerKeyboardNotifications];
+
+  [self setupScrollView];
+
+  MDCTextField *textFieldName = [[MDCTextField alloc] init];
+  textFieldName.translatesAutoresizingMaskIntoConstraints = NO;
+  [self.scrollView addSubview:textFieldName];
+
+  textFieldName.placeholder = @"Full Name";
+  textFieldName.delegate = self;
+  textFieldName.clearButtonMode = UITextFieldViewModeUnlessEditing;
+  textFieldName.backgroundColor = [UIColor whiteColor];
+
+  self.nameController = [[MDCTextInputControllerOutlinedField alloc] initWithTextInput:textFieldName];
+
+  MDCTextField *textFieldAddress = [[MDCTextField alloc] init];
+  textFieldAddress.translatesAutoresizingMaskIntoConstraints = NO;
+  [self.scrollView addSubview:textFieldAddress];
+
+  textFieldAddress.placeholder = @"Address";
+  textFieldAddress.delegate = self;
+  textFieldAddress.clearButtonMode = UITextFieldViewModeUnlessEditing;
+  textFieldAddress.backgroundColor = [UIColor whiteColor];
```

Comment:
These four lines seem to be repeated for each text field, consider writing a helper function?  It may help `-viewDidLoad` to feel a bit smaller.